Firmware/ultralcd.cpp

```diff
@@ -5272,6 +5272,7 @@ static void lcd_main_menu()
 
     // only allow starting SD print if hardware errors (temperature or fan) are cleared
     if(!get_temp_error()
+            && !printer_active()
```

Comment:
I feel `printingIsPaused()` is more appropriate here given the commit message. `printer_active` checks for a lot more than just paused print state.

Reply:
The `Print from SD` is shown during the power panic recovery while heating.
These menus should only be shown when the printer is "idle" or not active. This is working well while the it is printing and some other conditions now. Lot of these menus below check `!printer_active()` to NOT be shown.
   
Print from SD
Auto/Load filament
Preload to MMU
Load to nozzle
Unload filament
Eject from MMU
Cut filament

Settings
Calibration